fastlane_core/lib/fastlane_core/device_manager.rb

```diff
@@ -234,6 +219,47 @@ def launch(device)
 
         Helper.backticks("open -a #{simulator_path} --args -CurrentDeviceUDID #{device.udid}", print: $verbose)
       end
+
+      def copy_logs(device, log_identity, logs_destination_dir)
+        logs_destination_dir = File.expand_path(logs_destination_dir)
+        os_version = FastlaneCore::CommandExecutor.execute(command: 'sw_vers -productVersion', print_all: false, print_command: false)
+
+        if Gem::Version.new(os_version) >= Gem::Version.new('10.12.0')
+          copy_logarchive(device, log_identity, logs_destination_dir)
+        else
+          copy_logfile(device, log_identity, logs_destination_dir)
+        end
+      end
+
+      private
+
+      def copy_logfile(device, log_identity, logs_destination_dir)
+        logfile_src = File.expand_path("~/Library/Logs/CoreSimulator/#{device.udid}/system.log")
+        return unless File.exist?(logfile_src)
+
+        logfile_dst = File.join(logs_destination_dir, "system-#{log_identity}.log")
+
+        FileUtils.rm_rf(logfile_dst)
```

Comment:
`FileUtils.mkdir_p` is not necessary. It _was_ necessary in the `copy_logarchive` because an actual directory is being made